src/LoginRouter.js

```diff
@@ -126,6 +127,7 @@ export default BaseLoginRouter.extend({
     'signin/refresh-auth-state(/:token)': 'refreshAuthState',
     'signin/register': 'register',
     'signin/register-complete': 'registerComplete',
+    'signin/mfa-invalid-session': 'mfaInvalidSession',
```

Comment:
This is good, but what do you think if we make this more generic? Something like `signin/error` so this route can be reused if we ever need to. The controller is just the most basic controller that shows errors, nothing else... 
Thoughts? We can also cross this bridge once we need to (if we ever need to)

Reply:
That actually makes sense, and it was something i was thinking about.. I'll make this change